x-pack/docs/en/watcher/gs-index.asciidoc

```diff
@@ -1,6 +1,6 @@
 [role="xpack"]
-[[xpack-alerting]]
-= Alerting on cluster and index events
+[[watcher-alerting]]
+= Watching for cluster and index events
```

Comment:
This file looks inert. Maybe remove it entirely?

Also not sure why this isn't just `Watch for...`

Reply:
Will remove it. It's an artifact from when we built standalone getting started "books" for the translation effort.